packages/terra-consumer-nav/src/Nav.jsx

```diff
@@ -129,7 +129,7 @@ class Nav extends React.Component {
         >
           <Button icon={<IconClose />} className={cx('close-button')} onClick={() => { onRequestClose(); }} />
           <NavLogo {...logo} />
-          <NavItems navItems={navItems} />
+          <NavItems navItems={navItems} handleClick={onRequestClose} />
```

Comment:
will this cause problems if we ever have nav's open modals and thigns? If we won't do this today, then we can probably punt that problem

Reply:
it shouldn't be a big issue, i imagine the user would click the link which opens the modal but would also silently close the nav behind the open modal.